playground/vite.config.ts

```diff
@@ -43,6 +43,7 @@ export default defineConfig({
     Inspect({
       build: true,
       open: true,
+      hideDarkMode: false,
```

Comment:
We could use `embedded` instead

Reply:
Yeah I was suggesting the naming. `hideDarkMode` is a bit too specific, while for `embedded` in the future we could also add other features etc. Also we should go for `? embedded` in the query